ansible/group_vars/orange_pi_zero

```diff
@@ -0,0 +1,3 @@
+---
+nginx_worker_processes: 1
```

Comment:
`nginx_worker_processes` is normally equal to the CPU count. Given the OPi Zero has 4 CPUs, it would be appropriate to have `nginx_worker_processes: 4` (or a comment showing the reason why it isn’t 4)